app/controllers/admin/reports/metrics_controller.rb

```diff
@@ -1,40 +1,38 @@
 require 'csv'
 
-module Admin
-  module Reports
-    class MetricsController < Admin::ApplicationController
-      def index
-        @snapshots = Snapshot.order(created_at: :desc).page(params[:page]).per(20)
-      end
+class Admin::Reports::MetricsController < Admin::ApplicationController
+  def index
+    @snapshots = Snapshot.order(created_at: :desc).page(params[:page]).per(20)
+  end
 
-      def show
-        @snapshot = Snapshot.find(params[:id])
-      end
+  def show
+    @snapshot = Snapshot.find(params[:id])
+  end
 
-      def download
-        snapshot = Snapshot.find(params[:id])
-        send_data snapshot_to_csv(snapshot), filename: csv_filename(snapshot), type: 'text/csv'
-      end
+  def download
+    snapshot = Snapshot.find(params[:id])
+    send_data snapshot_to_csv(snapshot),
+              filename: csv_filename(snapshot),
+              type: 'text/csv'
+  end
 
-      private
+  private
 
-      def csv_filename(snapshot)
-        date = snapshot.created_at
-        "snapshot-#{date.year}-#{date.month}-#{date.day}.csv"
-      end
+  def csv_filename(snapshot)
+    date = snapshot.created_at
+    "snapshot-#{date.year}-#{date.month}-#{date.day}.csv"
```

Comment:
@paulanthonywilson put these in brackets, perhaps?

Reply:
Where do you mean? I think this looks clear as it is?

Reply:
@igelstorm we usually put parenthesis around parameters.

```
date = snapshot.created_at(
    "snapshot-#{date.year}-#{date.month}-#{date.day}.csv"
)
```

Reply:
I see what you mean. That's not how this code will be interpreted though: the string isn't an argument to `created_at`, it's a separate statement (an implicit `return`).

It's first assigning `date = snapshot.created_at`, and then returning a string with some attributes of `date` interpolated.

Reply:
Oh wow! That is so deceptive. In that case do you mind breaking the two statements with a line so that it doesn't look like `"snapshot-#{date.year}-#{date.month}-#{date.day}.csv"` is an argument for the `created_at` method?

#option 1: 
```
date = snapshot.created_at
return "snapshot-#{date.year}-#{date.month}-#{date.day}.csv"
```

#option 2: 
```
date = snapshot.created_at

"snapshot-#{date.year}-#{date.month}-#{date.day}.csv"
```

Reply:
Since it's already caused confusion, perhaps that should be made explicit — `return "snapshot..."`?

Reply:
I'm happy with either of the two options @Tr4pSt3R has suggested. I think it does need to be explicit that it isn't an argument as that was my first assumption.

Perhaps we could create an alias to the `created_at` method 
`date = snapshot.date_created`

Reply:
I prefer the blank line between the 2 statements, option 2. Its more in line with standards on other MAS repos.

I was not confused by the initial definition - a string was being returned and the snapshot date was being used to make up the string. Also the method name `csv_filename` indicates that probably a string is going to be returned.

Reply:
@margOnline you're right, looking at the full method block it is more clear. My first assumption and comment was based solely on looking at `.created_at` rather than the full method. 

I agree with option 2 being in line with MAS code.

Reply:
This is probably the ideal place to use the `.strftime` method since it specifically is made to address this kind of problem. However since the main point of this PR is to fix rubocop errors rather than a lot of refactoring perhaps the line break option is more sensible.